diff --git a/node_modules/leaflet/src/geometry/LineUtil.js b/node_modules/leaflet/src/geometry/LineUtil.js
index 24df8df..de6304b 100644
--- a/node_modules/leaflet/src/geometry/LineUtil.js
+++ b/node_modules/leaflet/src/geometry/LineUtil.js
@@ -236,12 +236,12 @@ export function _sqClosestPointOnSegment(p, p1, p2, sqDist) {
 // @function isFlat(latlngs: LatLng[]): Boolean
 // Returns true if `latlngs` is a flat array, false is nested.
 export function isFlat(latlngs) {
-	return !Util.isArray(latlngs[0]) || (typeof latlngs[0][0] !== 'object' && typeof latlngs[0][0] !== 'undefined');
+	return !L.LineUtil.isArray(latlngs[0]) || (typeof latlngs[0][0] !== 'object' && typeof latlngs[0][0] !== 'undefined');
 }
 
 export function _flat(latlngs) {
 	console.warn('Deprecated use of _flat, please use L.LineUtil.isFlat instead.');
-	return isFlat(latlngs);
+	return L.LineUtil.isFlat(latlngs);
 }
 
 /* @function polylineCenter(latlngs: LatLng[], crs: CRS): LatLng
